src/components/PackageRecommender.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState, useEffect } from "react";
+import Stack from "@mui/material/Stack";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import { Padding } from "@mui/icons-material";
+
+export default function PackageRecommender({ packageOne, packageTwo }) {
+  const [packageOneScore, setPackageOneScore] = useState(0);
+  const [packageTwoScore, setPackageTwoScore] = useState(0);
+  const [betterPackage, setBetterPackage] = useState("");
+  const [timesBetter, setTimesBetter] = useState(0);
+  const [packageData, setPackageData] = useState({});
+
+  useEffect(() => {
+    const scoreOne = calculateScore(packageOne);
```

Comment:
If you move these variables outside useEffect. None of the state would be required. 
Avoid using excessive states and useEffect if the logic can be handled without it.